helm/superset/values.yaml

```diff
@@ -216,6 +216,18 @@ ph-ee-engine:
     requests:
       cpu: "100m"
       memory: "256M"
+    ingress:
+      enabled: true
+      annotations:
+        kubernetes.io/ingress.class: "nginx"
+      tls:
+        - secretName: sandbox-secret
+      path: "/"
```

Comment:
all these variables should be added to template values.yaml as well ( with their default values)

Reply:
It is already added in the template values.yaml